### PR TITLE
Vebt 479 sundown sweep script

### PR DESCRIPTION
**Pull request by nfstern02**
## Summary

- *This work is behind a feature toggle (flipper): NO
- Added code to delete nightly work files for the VYE process.
- VEBT and we own maintaining this code.

## Related issue(s)

- [VEBT-479](https://jira.devops.va.gov/browse/VEBT-479)

## Testing done

- [x] *New code is covered by unit tests*
- Nightly work files were not being deleted in Amazon the S3 buckets we are using
- Run the job manually once to confirm the expected files are removed in Staging
- Push to Staging and run once manually to confirm script is behaving as expected.

## What areas of the site does it impact?
VYE (Verify Your Enrollment)

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [x]  Events are being sent to the appropriate logging solution
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x]  Feature/bug has a monitor built into Datadog (if applicable) N/A

### CODE REVIEW

**Review by GcioGregg — APPROVED** [review]

**Comment by ryan-mcneil** [comment]
@nfstern02 Is it worth adding any tests around the new error handling you added? Thanks!

**Comment by nfstern02** [comment]
> @nfstern02 Is it worth adding any tests around the new error handling you added? Thanks!

I will look into it. I'm going to mock them and I hope that works for you.

**Comment by nfstern02** [comment]
Added tests for exception handling and logging.

File: modules/vye/spec/sidekiq/vye/sundown_sweep_spec.rb [hunk 1]
```diff
@@ -19,4 +19,64 @@
...
+    let(:s3_client) { Aws::S3::Client.new(stub_responses: true) }
+    let(:logger) { Rails.logger }
+
+    before do
+      allow(Aws::S3::Client).to receive(:new).and_return(s3_client)
+      allow(logger).to receive(:error)
+    end
+
+    it 'throws an exception when the bucket cannot be found on AWS' do
+      s3_client.stub_responses(:delete_object, 'NoSuchBucket')
```
Comment (ryan-mcneil) [inline modules/vye/spec/sidekiq/vye/sundown_sweep_spec.rb:35]: I pulled it down to confirm, but these `stub_responses` lines aren't needed. The following line is already saying that "whenever `s3_client.delete_object` is called, raise ERROR". So I think you can just remove those 4 statements

Comment (nfstern02) [reply modules/vye/spec/sidekiq/vye/sundown_sweep_spec.rb:35]: Let me take a look.

Comment (nfstern02) [reply modules/vye/spec/sidekiq/vye/sundown_sweep_spec.rb]: That was correct and I removed them and pushed. Thx for the knowledge drop.

**Review by ryan-mcneil — APPROVED** [review]

**Pull request merged**